app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -266,33 +268,46 @@ fun MessageExpireLabel(messageContent: UIMessageContent?, timeLeft: String) {
 
         is UIMessageContent.AssetMessage -> {
             StatusBox(
-                statusText = if (messageContent.downloadStatus == Message.DownloadStatus.SAVED_INTERNALLY
-                    || messageContent.downloadStatus == Message.DownloadStatus.SAVED_EXTERNALLY
-                ) stringResource(
-                    R.string.self_deleting_message_time_left,
-                    timeLeft
-                )
-                else stringResource(R.string.self_deleting_message_label, timeLeft)
+                statusText = if (messageContent.downloadStatus == Message.DownloadStatus.SAVED_INTERNALLY ||
```

Comment:
This if is being used in multiple places and if I understand correctly forgetting about SAVED_EXTERNALLY was an issue, so maybe we could create an extension function to make it easier, like `fun DownloadStatus.isSaved(): Boolean`?